spec/system/claims/view_organisations_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+feature "View organisations as a multi org user" do
```

Comment:
Should this be a **system test** rather than a **feature test**?

```suggestion
RSpec.describe "View organisations", type: :system do
```